libcaf_core/caf/ipv4_address.hpp

```diff
@@ -46,6 +46,8 @@ class ipv4_address : public byte_address<ipv4_address> {
 
   explicit ipv4_address(array_type bytes);
 
+  /// Construct an ipv4 address from the network representation of the address
```

Comment:
To me, `network representation` is very vague and an unusual wording. I'd prefer the much more common term "network byte order". Also, this function is not meant for public use. Hence, please add `@private`.

Reply:
Why is this a private API? I'm using it to parse addresses from pcap files. (I'm sure those are only v4.) What would be the alternative here? Deconstructing the int into four octets or the string representation seems like an unnecessary work-around.

Reply:
It's private simply by extension, because the `bits(uint32_t)` setter is private. I don't have strong feeling towards making both public, though. Im finde with removing that restriction as long as you update the documentation of the `bits` getter and setter with mentioning "network byte order".

Reply:
:+1: I'll update the comments.